sympy/geometry/tests/test_ellipse.py

```diff
@@ -477,6 +477,10 @@ def test_section_modulus_and_polar_second_moment_of_area():
     assert e.section_modulus() == (18*pi, 6*pi)
     assert e.polar_second_moment_of_area() == 120*pi
 
+    e = Ellipse(Point(0, 0), 2, 2)
+    assert e.section_modulus() == (2*pi, 2*pi)
+    assert e.section_modulus(Point(2, 2)) == (2*pi, 2*pi)
```

Comment:
You could test that this method accepts a 2-element tuple with `e.section_modulus((2,2))`...after making the change to Ellipse.section_modulus:
```
        else:
            # taking x and y as distances of the given point from the center
            point = Point(point)
            y = point.y - y_c
```

Reply:
That is a nice suggestion, done!

Reply:
@smichr  I tried this suggestion (commit above) but it altered something in the output, it passes the tests on local but not on Travis CI, could you take a look at it and tell me if you see anything wrong?